src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumn.js

```diff
@@ -167,6 +167,9 @@ class ProjectColumn extends Component {
         </div>
         <div className={css(styles.columnBody)}>
           <div className={css(styles.columnInner)}>
+            {(projects.length === 0 && label === 'done') &&
```

Comment:
Heh, that's my absolute _least_ favorite option. Here's why: In an effort to make sure our UI doesn't "get in the way" of getting work done, I think we should make sure that text only appears when the user wants to read it. Having text hang out makes for a dirty experience & so in the goal of customer retention, i'd say this is dangerous because if i'm gonna abandon a product, i'm gonna do it waaaay sooner than 5 meetings. if i love the product, i'm gonna make a fake done item just to get rid of it. (UX vs UI and all that junk).

I think this message is useful after 2 events:
- onboarding
- just after my meeting

at all other times, i just want to be left alone.

if i haven't convinced you we can still press on, just wanted to share.